### PR TITLE
TEXTURE_2D_ARRAY for EXT_EGL_image_array

### PR DESCRIPTION
**Pull request by stonesthrow**
Closes: https://github.com/KhronosGroup/OpenGL-Registry/issues/496

### CODE REVIEW

**Review by oddhack — APPROVED** [review]

File: extensions/EXT/EXT_EGL_image_array.txt [hunk 1]
```diff
@@ -32,7 +33,7 @@ Number
 
 Dependencies
 
-    OpenGL ES 2.0 is required.
+    OpenGL ES 2.0 affects the definition of this extension
```
Comment (pdaniell-nv) [inline extensions/EXT/EXT_EGL_image_array.txt:36]:
```suggestion
    OpenGL ES 2.0 is required
    OpenGL ES 3.0 affects the definition of this extension
```

File: extensions/EXT/EXT_EGL_image_array.txt [hunk 2]
```diff
@@ -65,6 +66,11 @@ Additions to Chapter 3 of the OpenGL ES 2.0 Specification
 
         "Currently, <target> must be TEXTURE_2D or TEXTURE_2D_ARRAY."
 
+Interactions with OpenGL ES 2.0
```
Comment (pdaniell-nv) [inline extensions/EXT/EXT_EGL_image_array.txt:69]:
```suggestion
Interactions with OpenGL ES 3.0  
```

**Review by pdaniell-nv — CHANGES_REQUESTED** [review]

**Comment by pdaniell-nv** [comment]
Merge approved once these little issues are resolved.

**Comment by stonesthrow** [comment]
Thanks Piers, updated, PTAL.

**Review by pdaniell-nv — APPROVED** [review]

**Comment by pdaniell-nv** [comment]
@oddhack this is approved to merge.

**Pull request merged**